ajax/libs/reactstrap/package.json

```diff
@@ -24,7 +24,7 @@
     {
       "basePath": "dist",
       "files": [
-        "reactstrap.min.js*"
+        "**/reactstrap!(*es*|*cjs*)"
```

Comment:
Should better be `"**/reactstrap!(*.es.*|*.cjs.*)"`?

Reply:
Updated and rebased.